README.md

```diff
@@ -3,11 +3,23 @@
 This is the source code for [https://amplifyingfsharp.io/](https://amplifyingfsharp.io/).  
 If you are looking to submit a session you can visit [this page](https://amplifyingfsharp.io/join-us/) instead.
 
-## Requirements
+## Donate
 
-You need Node.JS 20 LTS or equivalent to run this project.
+[![Open Collective donate button](https://opencollective.com/amplifying-fsharp/donate/button.png?color=blue)](https://opencollective.com/amplifying-fsharp/donate)
 
-## 🧞 Commands
+We have an [Open Collective](https://opencollective.com/amplifying-fsharp#category-ABOUT) and thank
```

Comment:
We could update the last part ? ie `We have an [Open Collective](https://opencollective.com/amplifying-fsharp#category-ABOUT). Thank you to all our donors! `

Reply:
Sure! Also, learn the suggestion thing: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

![image](https://github.com/amplifying-fsharp/amplifying-fsharp.github.io/assets/2621499/551a1c34-0ba4-418a-88e5-69d54aa7c755)